tools/setup_riak

```diff
@@ -2,11 +2,18 @@
 
 sleep 20
 
-RIAK_HOST="http://localhost:8098"
+RIAK_HOST=${RIAK_HOST:-"localhost"}
+RIAK_PORT=${RIAK_PORT:-"8098"}
+RIAK_HOST="http://${RIAK_HOST}:${RIAK_PORT}"
+
+VCARD_SCHEMA_PATH=${VCARD_SCHEMA_PATH:-"tools/vcard_search_schema.xml"}
```

Comment:
What about naming this variable like: `RIAK_SEARCH_VCARD_SCHEMA`? Not necessary exactly like that, but I think the `RIAK_` prefix alone would be really nice to see (like in other variables). Those files are specific to `Riak Search` service so if such variable is set in different file, it would be nice to know what service it's referencing to. Especially considering that other variables have `RIAK_` prefix, so if this would also had one, it would be obvious that it's used in same file/service.